modules/internal/ChapelSerializedBroadcast.chpl

```diff
@@ -23,7 +23,7 @@
 //
 module ChapelSerializedBroadcast {
   use ChapelLocale;
-  use CPtr;
+  public use CPtr;
```

Comment:
I would imagine we don't want CPtr to be visible by default. Does this need some sort of TODO comment?

Reply:
Also noted in https://github.com/Cray/chapel-private/issues/1184